components/src/main/java/uk/gov/hmrc/components/extensions/ViewExt.kt

```diff
@@ -23,7 +23,7 @@ import androidx.core.view.ViewCompat
 import androidx.core.view.accessibility.AccessibilityNodeInfoCompat
 import uk.gov.hmrc.components.R
 
-fun View.setAccessibilityMessage(message: CharSequence) {
+fun View.setAccessibilityMessage(message: CharSequence, className: String?) {
```

Comment:
Can we add a default parameter here so it's not a breaking change and we don't need to update all our existing usages?
```suggestion
fun View.setAccessibilityMessage(message: CharSequence, className: String? = null) {
```